CHANGELOG.unreleased.md

```diff
@@ -0,0 +1,24 @@
+# Unreleased changes
+
+Add here any changes made in a PR that are relevant to end users. Allowed sections:
+
+* Fixed
+* Changed
+* Added
+* Deprecated
```

Comment:
```suggestion
Added for new features.
* Changed  - for changes in existing functionality.
* Deprecated - for soon-to-be removed features.
* Removed - for now removed features.
* Fixed - for any bug fixes.
* Security -  in case of vulnerabilities.
```
from the keepachangelog...

Reply:
good idea, thanks!